mobile/lib/screens/dashboard/dashboard_view.dart

```diff
@@ -598,6 +598,7 @@ class _DashboardViewState extends State<DashboardView>
       },
       onError: (error) {
         debugPrint('error listening to location updates : $error');
+        logException(error, null);
```

Comment:
Hi @jordanopos do we need to "await" here too?

Reply:
Hey @Baalmart , the function is designed to handle requests asynchronously, so we don't need to wait for it to finish before we can move on.

Reply:
thanks @jordanopos